android/app/src/main/java/org/fossasia/openevent/core/speaker/SpeakersListAdapter.java

```diff
@@ -143,8 +146,20 @@ else if (!TextUtils.isEmpty(speakerData) && (sortType == 1 || sortType == 2))
     public String getSectionTitle(int position) {
         //this String will be shown in a bubble for specified position
         Speaker speaker = getItem(position);
-        String name = speaker.getName();
-        return name.substring(0, 1);
+        sortType = SharedPreferencesUtil.getInt(ConstantStrings.PREF_SORT_SPEAKER, 0);
+        String title;
+        if (sortType == 0) {
+            title = speaker.getName();
+        } else if (sortType == 1) {
+            title = speaker.getOrganisation();
+        } else {
+            title = speaker.getCountry();
+        }
+
+        if (title.length() == 0 || title.isEmpty()){
+            return "-";
+        }
+        return String.valueOf(title.charAt(0));
```

Comment:
What's 0 and 1?

Reply:
are you asking about the sortOrder?
Those are the sort orders used in the bindView method. 
I was looking for a constant but there was none.

Reply:
There is, and if it isn't please create these

Reply:
@iamareebjamal done